tests/ubpf_execution.rs

```diff
@@ -3226,69 +3224,53 @@ fn test_tcp_sack_nomatch() {
 }
 
 #[test]
-fn test_large_program() {
-    let mut insns = vec![ebpf::Insn::default(); ebpf::PROG_MAX_INSNS * 2 - 1];
-    for index in (0..(ebpf::PROG_MAX_INSNS * 2 - 1)).step_by(2) {
-        insns[index].opc = ebpf::LD_DW_IMM;
-    }
-    insns[ebpf::PROG_MAX_INSNS - 4].opc = ebpf::JA;
-    insns[ebpf::PROG_MAX_INSNS - 4].off = 9;
-    insns[ebpf::PROG_MAX_INSNS - 3].opc = ebpf::MOV64_IMM;
-    insns[ebpf::PROG_MAX_INSNS * 2 - 2].opc = ebpf::EXIT;
-
-    let config = Config {
-        enable_instruction_tracing: true,
-        ..Config::default()
-    };
-
-    let mut program = insns
-        .iter()
-        .map(|insn| insn.to_vec())
-        .flatten()
-        .collect::<Vec<_>>();
-    #[allow(unused_mut)]
-    {
-        let mut bpf_functions = BTreeMap::new();
-        register_bpf_function(&mut bpf_functions, 0, "entrypoint").unwrap();
-        let mut executable = <dyn Executable<UserError, TestInstructionMeter>>::from_text_bytes(
-            program.as_slice(),
-            bpf_functions,
-            None,
-            config,
-        )
-        .unwrap();
-        test_interpreter_and_jit!(
-            executable,
-            [],
-            (),
-            { |_vm, res: Result| res.unwrap() == 0x0 },
-            ebpf::PROG_MAX_INSNS as u64 - 4
-        );
-    }
+fn test_ja_to_lddw() {
+    // test_interpreter_and_jit_asm!(
```

Comment:
@Lichtso Can you take a look at this test?  Both fail but the analysis/trace produces two different results (extra `ja` in the case of the jit.

Reply:
Sure, I will take a look.